docs/index.html

```diff
@@ -15,85 +15,34 @@
 
 <section class="block block-usecase block--centered">
   <div class="centered-block">
-    <p>Top and bottom containers may fit any content provided to Sui-Card. Use the following classes to layout styling:</p>
-    <ul>
-      <li>
-        <strong>sui-Card-topComponent:</strong>
-        <p>This is the top placeholder where you can fit any custom component: from a simple picture to a full featured slider.</p>
-        <p>In the example below an anchor element wrapping an image is stored in constant 'top' and passed to SUI-Card component as 'top-content'.</p>
-      </li>
-      <li>
-        <strong>sui-Card-bottomComponent:</strong>
-        <p>This is the bottom placeholder. Pass a custom component or insert your static HTML code as shown in example.</p>
-      </li>
-    </ul>
 
-    <h2>SUI-Card Component implementation</h2>
-    <div>
-      <pre>
-        <code class="language-js">
-          &lt;SuiCard
-            // Implementation passing topComponent
-            topComponent={top}
-            bottomComponent={bottom}
-
-            // Set to true to change orientation
-            landscapeLayout={false}
-            /&gt;);
-        </code>
-      </pre>
-    </div>
-    <h2>SUI-Card Portrait render Example</h2>
-    <div id="mainPortrait"></div>
-
-    <p>If no topComponent property is passed a default picture with link can be set using the following props:</p>
+    <h2>SUI-Card render Example</h2>
+    <div id="main"></div>
 
     <h2>SUI-Card Component implementation</h2>
     <div>
-      <pre>
-        <code class="language-js">
-          &lt;SuiCard
-            // Implementation passing default values
-            topImgDefault='http://bit.ly/1LYWHy0'
-            topImgDefaultLink='http://bit.ly/1KPfv2H'
-
-            bottomComponent={bottom}
-
-            // Set to false to change default orientation
-            landscapeLayout={true}
-            contentFirst={true}
-            /&gt;);
-        </code>
-      </pre>
+<pre>
+  <code class="language-js">
+    const data = {
+      multimedia: [
+        {
+          src: 'https://scontent-mad1-1.cdninstagram.com/t51.2885-15/e15/11189686_499366993548088_1592806536_n.jpg',
+          alt: 'Property picture',
+          type: 'image',
+          link: 'https://www.instagram.com/p/UCYp_ypMkN/?taken-by=davecarter'
+        }
+      ],
+      title: 'Bicing Old town Barcelona',
+      description: 'lorem ipsum'
+    };
+
+    ReactDom.render(
+      &lt;SuiCard data={data} /&gt;,
+      document.getElementById('main'));
```

Comment:
Going a little further. I would like to add back a missed functionality that we had when using React v.0.13: Live preview with [Playground Component](http://stack.formidable.com/component-playground/).

We had compatibility issues with React 0.14. That's why we only have this syntax highlighting using `prism.js` It would be nice to implement back live preview.